GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -296,12 +296,12 @@ gdjs.RuntimeGamePixiRenderer.prototype.bindStandardEvents = function(manager, wi
         manager.onMouseMove(pos[0], pos[1]);
     };
     renderer.view.onmousedown = function(e){
-        manager.onMouseButtonPressed(e.button === 2 ? 1 : 0);
+        manager.onMouseButtonPressed(e.button === 2 ? 1 : 0 || e.button === 1 ? 2 : 0);
         if (window.focus !== undefined) window.focus();
         return false;
     };
     renderer.view.onmouseup = function(e){
-        manager.onMouseButtonReleased(e.button === 2 ? 1 : 0);
+        manager.onMouseButtonReleased(e.button === 2 ? 1 : 0 || e.button === 1 ? 2 : 0);
```

Comment:
Same the combination of the ternary operator and the || operator seems incorrect, better use `e.button === 2 ? 1 : e.button === 1 ? 2 : 0`